python/pyspark/sql/tests/connect/test_connect_function.py

```diff
@@ -412,6 +412,125 @@ def test_aggregation_functions(self):
             sdf.groupBy("a").agg(SF.percentile_approx(sdf.b, [0.1, 0.9])).toPandas(),
         )
 
+    def test_date_ts_functions(self):
```

Comment:
if `toPandas` doesn't work correctly with timestamp, you can use the newly added comparison method [compare_by_show](https://github.com/apache/spark/blob/master/python/pyspark/sql/tests/connect/test_connect_function.py#L66)

Reply:
Sounds good, thanks!